### ISSUE TITLE
Inconsistent search fields

### ISSUE THREAD

**Issue by flopp999** [grafana/grafana#22934]
**Steps to reproduce**:
Open all search fields

**Actual Result**:
I found that that search fields are not consistent

**Expected Result**
i thought that all search fields should look the same

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 6.7
- Data source type & version:influx latest
- User OS & Browser: windows, chrome

With all my comments I have used number 1 as default

1. When clicked on Dashboard - Manage from left menu
- this is my default
![image](https://user-images.githubusercontent.com/21694965/77179234-7a88a380-6ac8-11ea-979d-f58a45a34b69.png)

2. Search dashboard when clicked on top left
-not a big D in dashboards and with an s at the end
-not a black square
-Filter by is in another "window" and also a search field
![image](https://user-images.githubusercontent.com/21694965/77179265-870cfc00-6ac8-11ea-92c7-bc22b1d1a837.png)

3. Dashboard - Playlist create playlist
-missing the small magnifier
-filter by is totally different and without a search box
![image](https://user-images.githubusercontent.com/21694965/77179171-6644a680-6ac8-11ea-93ba-794e660be7ad.png)

4. Alerts search
-starts with "Search" the above have "Find" as start word
-add "by name" so it would be "Search alerts by name"?
![image](https://user-images.githubusercontent.com/21694965/77179375-b28fe680-6ac8-11ea-8cd9-22e3b0562cc5.png)

5. Configuration - Data Source
-starts with "Filter"
-I tried to search with type but I couldn't find out that worked
![image](https://user-images.githubusercontent.com/21694965/77179432-c76c7a00-6ac8-11ea-8f5c-0dc635f2df9e.png)

6.  Configuration - Users
-starts with "Filter"
![image](https://user-images.githubusercontent.com/21694965/77179497-db17e080-6ac8-11ea-9bf8-b26e1e44f000.png)

7. Configuration - Plugins
-starts with "filter"
![image](https://user-images.githubusercontent.com/21694965/77179564-eec34700-6ac8-11ea-9a18-0afdebb1d552.png)

8. Configuration - API Keys
-starts with "Search"
-add "API" in front of "keys"?
![image](https://user-images.githubusercontent.com/21694965/77179804-43ff5880-6ac9-11ea-8c53-d256e62016f9.png)

9. Admin - Users
-this could be same text as number 6?
![image](https://user-images.githubusercontent.com/21694965/77179775-3649d300-6ac9-11ea-8dc7-ef38c81ca0f0.png)

I will change all text so they get more consistent in a PR.
But I have no knowledge about the filtering window or feature maybe someone else know this and can made a PR?

**Comment by aknuds1** [comment]
Can you please fill in also the initial fields of the standard issue template? Thanks!

**Comment by flopp999** [comment]
> Can you please fill in also the initial fields of the standard issue template? Thanks!
like this?

**Comment by aknuds1** [comment]
Just follow the standard issue template, which is there when you create a new issue:

**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version:
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

**Comment by flopp999** [comment]
When I create a new Issue I get this template, I choose Accessibility issue


**Steps to reproduce**:

**Actual Result**:

**Expected Result**

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by aknuds1** [comment]
Ah sorry, I wasn't aware of the accessibility issue template. In that case, it looks as if you've used the right template! Thanks!

**Comment by dprokop** [comment]
Fixed by #22929
